components/patient-data/rest/src/test/java/org/phenotips/data/rest/internal/DefaultPatientByExternalIdResourceImplTest.java

```diff
@@ -148,16 +156,21 @@ public void setUp() throws ComponentLookupException, URISyntaxException
         Provider<XWikiContext> provider = this.mocker.getInstance(XWikiContext.TYPE_PROVIDER);
         this.context = provider.get();
         ReflectionUtils.setFieldValue(this.component, "uriInfo", this.uriInfo);
+        ReflectionUtils.setFieldValue(this.component, "currentResolver", this.resolver);
```

Comment:
This isn't needed, since it's an injected dependency, you can get the automocked instance with `mocker.getInstance`.